autogpt_platform/frontend/src/components/monitor/FlowRunInfo.tsx

```diff
@@ -1,79 +1,158 @@
-import React, { useCallback } from "react";
-import AutoGPTServerAPI, { GraphMeta } from "@/lib/autogpt-server-api";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import AutoGPTServerAPI, {
+  BlockIORootSchema,
+  Graph,
+  GraphMeta,
+  NodeExecutionResult,
+} from "@/lib/autogpt-server-api";
 import { FlowRun } from "@/lib/types";
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
 import Link from "next/link";
 import { Button, buttonVariants } from "@/components/ui/button";
 import { IconSquare } from "@/components/ui/icons";
-import { Pencil2Icon } from "@radix-ui/react-icons";
+import { ExitIcon, Pencil2Icon } from "@radix-ui/react-icons";
 import moment from "moment/moment";
 import { FlowRunStatusBadge } from "@/components/monitor/FlowRunStatusBadge";
+import RunnerOutputUI, { BlockOutput } from "../runner-ui/RunnerOutputUI";
 
 export const FlowRunInfo: React.FC<
   React.HTMLAttributes<HTMLDivElement> & {
     flow: GraphMeta;
     flowRun: FlowRun;
   }
 > = ({ flow, flowRun, ...props }) => {
+  const [isOutputOpen, setIsOutputOpen] = useState(false);
+  const [blockOutputs, setBlockOutputs] = useState<BlockOutput[]>([]);
+  const api = useMemo(() => new AutoGPTServerAPI(), []);
+
+  const fetchBlockResults = useCallback(async () => {
+    console.log("fetchBlockResults");
+    const graph = await api.getGraph(flow.id, flow.version);
```

Comment:
`/graphs/{graph_id}/executions/{graph_exec_id}` has block_id as part of its response. I don't think we need to do nested requests for graph here?

Reply:
You are right, ts type was missing `block_id`, added